lib/screens/register_screen.dart

```diff
@@ -17,181 +20,272 @@ class RegisterScreen extends StatefulWidget {
-        body: ModalProgressHUD(
-          color: kPrimaryColor,
-          inAsyncCall: this._showSpinner,
-          child: Material(
-            child: Container(
-              color: kBackgroundColor,
-              child: Center(
-                child: NotificationListener<OverscrollIndicatorNotification>(
-                  onNotification: (OverscrollIndicatorNotification overscroll) {
-                    overscroll
-                        .disallowGlow(); // This will stop overscroll glow effect.
-                    return;
-                  },
-                  child: ListView(
-                    padding:
-                        EdgeInsets.symmetric(horizontal: 50.0, vertical: 10.0),
-                    shrinkWrap: true,
-                    children: <Widget>[
-                      Center(
-                        child: Stack(
-                          children: <Widget>[
-                            Text(
-                              'Start',
-                              style: kHeadingStyle,
+    return ChangeNotifierProvider<AddressModel>(
+      create: (context) => AddressModel(),
+      child: SafeArea(
+        child: Scaffold(
+          resizeToAvoidBottomInset: true,
+          body: ModalProgressHUD(
+            color: kPrimaryColor,
+            inAsyncCall: this._showSpinner,
+            child: Material(
+              child: Container(
+                color: kBackgroundColor,
+                child: Padding(
+                  padding: const EdgeInsets.symmetric(horizontal: 50.0),
+                  child: Center(
+                    child:
+                        NotificationListener<OverscrollIndicatorNotification>(
+                      onNotification:
+                          (OverscrollIndicatorNotification overscroll) {
+                        // This will stop overscroll glow effect.
+                        overscroll.disallowGlow();
+                        return;
+                      },
+                      child: ListView(
+                        shrinkWrap: true,
+                        children: <Widget>[
+                          Center(
+                            child: Stack(
+                              children: <Widget>[
+                                Text(
+                                  'Start',
+                                  style: kHeadingStyle,
+                                ),
+                                Padding(
+                                  padding: const EdgeInsets.only(top: 50.0),
+                                  child: Row(
+                                    crossAxisAlignment:
+                                        CrossAxisAlignment.baseline,
+                                    textBaseline: TextBaseline.alphabetic,
+                                    children: <Widget>[
+                                      Text(
+                                        'Sharing',
+                                        style: TextStyle(
```

Comment:
Use `kHeadingStyle` with `.copywith()` instead of rewriting the `TextStyle` individually.

Reply:
Done.